config.js

```diff
@@ -12,7 +12,8 @@ const config = {
   moreInfoTitle: 'For more information about the issues follow the links: \n',
   tslintProjectWebsite: 'https://github.com/webschik/tslint-config-security',
   numFilesPerPage: 30,
-  numAnnotationsPerUpdate: 50
+  numAnnotationsPerUpdate: 50,
+  configFilePath: '.precaution.json'
```

Comment:
Yes, but JavaScript standard doesn't allow us this.
The build of Travis CI will fail because of that.

Reply:
Yes, but the JavaScript standard doesn't allow us this.
The build of Travis CI will fail because of that.

Reply:
I do it all the time in javascript.

Reply:
Yes, it's allowed.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Trailing_commas

Reply:
I mean [JavaScript standard](https://standardjs.com/) guidelines we are following.
I get a style error
![image](https://user-images.githubusercontent.com/16246778/57062089-d0ddcf80-6cc7-11e9-9649-9e5f2c44e1d0.png)

and when I run `npm test` I get this: 
![image](https://user-images.githubusercontent.com/16246778/57062095-daffce00-6cc7-11e9-89f7-dd38c309e740.png)

which means that the build will fail because of that.

Reply:
Ah, I see.  Well, I recommend we remove that check from eslint.  Comma dangles are very useful for source control diffs.

Reply:
Oh this is standardjs, which doesn't allow config.

Reply:
I tried using standard on another project and immediately ran into this bug that has gone unfixed for 10 months.  So my confidence on using standard is not good as a result.

https://github.com/standard/standard/issues/1165